StudCity.Frontend/src/app/features/userSlice.js

```diff
@@ -20,12 +20,20 @@ const initialState = {
 			backgroundImage: '',
 		},
 	},
+	searchData: {
```

Comment:
Lets rename this property to `users: {...}` . What do you think? It will be more readable.